app/main/util/dto.py

```diff
@@ -166,3 +166,19 @@ def getFileParser(cls, loc='files'):
         imgGen.add_argument('file', location=loc, required=True)
 
         return imgGen
+
+
+class StoryDto:
+    api = Namespace('story', description='For story card related operations')
+    story = api.model('story', {
+        'title': fields.String(required=True),
+        'article_summary': fields.String(required=True),
+        'image_link': fields.String(required=True),
+        'date': fields.String(required=True),
+        'reading_time': fields.Integer(required=True)
+    })
+
+    queryParams = api.model('queryParams', {
+        'offset': fields.Integer(required=True),
+        'limit': fields.Integer(required=True)
```

Comment:
Why is offset and limit a required field? It shouldn't be required.

Reply:
Actually that DTO model itself isn't required. Swagger doesn't allow GET requests to have a body so we'll have to pass the params in the URL itself.